manifests/autotune-configs/container-config.yaml

```diff
@@ -34,3 +34,17 @@ tunables:
   - response_time
   - throughput
   - resource_usage
+
+- name: someRequest
+  value_type: categorical
+  categorical_values:
+    - one
+    - two
+  step: 0.01
+  queries:
+    - datasource: 'prometheus'
+        query: '(container_cpu_usage_seconds_total{$CONTAINER_LABEL$!="POD", $POD_LABEL$="$POD$"}[1m])'
+  slo_class:
+    - response_time
+    - throughput
+    - resource_usage
```

Comment:
Please use the right tunables instead of dummy ones. You can find the entire list of tunables from here -> https://github.com/kruize/autotune-results/blob/main/techempower/experiment-12/benchmark.yaml